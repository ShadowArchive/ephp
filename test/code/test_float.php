<?php

$data = .11;
var_dump($data);

$data += .22 + .33;
var_dump($data);

