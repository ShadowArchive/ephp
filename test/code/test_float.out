float(0.11)
float(0.66)
